examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -4,24 +4,23 @@
  */
 import { SharedMap } from "@fluid-experimental/fluid-framework";
 import { FrsClient, FrsConnectionConfig } from "@fluid-experimental/frs-client";
-import { TinyliciousClient } from "@fluid-experimental/tinylicious-client";
 import { DiceRollerController } from "./controller";
 import { ConsoleLogger } from "./ConsoleLogger";
 import { renderAudience, renderDiceRoller } from "./view";
 
 // Define the server we will be using and initialize Fluid
 const useFrs = process.env.FLUID_CLIENT === "frs";
 
-const connectionConfig: FrsConnectionConfig = {
+const connectionConfig: FrsConnectionConfig = useFrs ? {
```

Comment:
As we discussed

```typescript
const connectionConfig: FrsConnectionConfig = isLocal ? {
    tenantId: "local",
    tokenProvider: new LocalTokenProvider({userconfig...}),
    orderer: "http://localhost:7070",
    storage: "http://localhost:7070",
}: {
    tenantId: "FooBar"
    tokenProvider: new AzureFunctionTP(...config...)
    orderer: "...",
    storage: "...",
}
```

Reply:
Updated to have matching interfaces for both local and remote as we discussed. Until we have the `AzureFunctionTP`, it is currently using an `InsecureTokenProvider` that takes in the key

Reply:
Can this project import `test-runtime-utils` in the interim instead of re-exporting through `frs-client` then?  Re-exporting through `frs-client` seems a little odd

Reply:
This would also mean though that any 3P app that is consuming the FrsClient will also have to import the `InsecureTokenProvider` from a separate package. Keeping them together allows for a single import, instead of directing the devs to another package they will need